packages/node_modules/@node-red/nodes/core/io/21-httprequest.html

```diff
@@ -102,6 +107,7 @@
             paytoqs: {value: false},
             url:{value:"",validate:function(v) { return (v.trim().length === 0) || (v.indexOf("://") === -1) || (v.trim().indexOf("http") === 0)} },
             tls: {type:"tls-config",required: false},
+            "http-persistent":{value:true},
```

Comment:
Can we rename this to just `persist`? Prefer to keep the node properties as short as is reasonable to minimise the impact on flow file size.

Reply:
Sure, please go ahead.

Reply:
(I think he meant you ;-)

Reply:
> 
> 
> Can we rename this to just `persist`? Prefer to keep the node properties as short as is reasonable to minimise the impact on flow file size.

Done